pkgs/by-name/op/openterface-qt/package.nix

```diff
@@ -4,9 +4,21 @@
   makeDesktopItem,
   copyDesktopItems,
   fetchFromGitHub,
+  writeText,
   qt6,
   libusb1,
 }:
+let
+  udevRules = writeText "60-openterface.rules" ''
```

Comment:
Can we comment the source of this?

Reply:
Do you mean do something like link to the upstream instructions, as in this PR? https://github.com/TechxArtisanStudio/Openterface_QT/blob/b1ed50672913ab30dc64e93be121647974403316/README.md#for-linux-users

I can do that, although the rules are different - the device IDs are acquired by my own testing.

Reply:
Yep. So that the next person could replicate the text without having to research everything on their own :)